src/content/energy_consumption/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Ethereum Energy Consumption
+description: The basic information you need to understand Ethereum's energy consumption.
+lang: en
+sidebar: true
+---
+
+# Introduction to Ethereum's Energy Consumption {#introduction-to-ethereum-energy-consumption}
+
+Since inception, sustainability has been a core value for Ethereum. However, until recently sustainability has sat uncomfortably alongside two other core values: security and scaleability. This page will explain why building out Ethereum to its current state has required a significant upfront carbon cost and why Ethereum is now on the cusp of becoming a truly low-carbon blockchain without undermining its other core principles.
+
+## Energy secures the network {energy-secures-the-network}
+
+Transactions on the Ethereum blockchain are validated by miners. Miners are node operators who bundle together transactions from the mempool into ordered blocks, which are added to the Ethereum blockchain. The new blocks are broadcast to all the other node operators who run the transactions independently and verify that the checksum generated by the miner proposing the block agrees with their own. Since the checksum is generated based on the entire history of the chain, any dishonesty shows up as an inconsistency in the checksums generated by different nodes. Assuming the checksums are verified, the block is added to the blockchain on every node. If the checksums do not match up, the blockchain forks, and after a few blocks the valid branch is assumed to be the one with the longest valid chain.
+
+This only works if there is a cost associated with minign and some unpredictability about which specific node submits the next block. These conditions are met by imposing proof-of-work (PoW). To be eligible to submit a block of transactions a miner must solve an arbitrary computational puzzle faster than any other miner. This creates competition between miners and cost in the form of energy expenditure. In order to successfully defraud the blockchain, a dishonest node operator would have to consistently win the race to solve the PoW puzzle which is both very unlikely and prohibitively expensive. PoW is therefore a way to secure the network. By requiring proof of computational work, PoW underpins the longest-chain consensus mechanism that ensures validity of the blockchain.
+
+Ethereum has implemented the PoW protocol since its inception. While migration away from PoW has always been a fundamental goal of Ethereum, it has also arguably been the most philosophically and technologically challenging because the viable alternatives all required some compromise in one of Ethereum's core principles.
```

Comment:
This is good but should be tackled from a slightly higher level. Seems like it intends to be directed to beginners but them makes assumptions about mempools and blocks. For learn pages, where possible, I'd lean towards not getting into the technicals too much.